### PR TITLE
test-configs.yaml: Add simple LTP test plan

### PR DESCRIPTION
**Pull request by Lakshmipathi**
Signed-off-by: Lakshmipathi <lakshmipathi.ganapathi@collabora.com>

### CODE REVIEW

**Comment by Lakshmipathi** [comment]
LAVA job run# https://lava.collabora.co.uk/scheduler/job/2574293/definition

**Comment by gctucker** [comment]
This is causing some errors in the Jenkins `test-runner` job:
```
Template not found: templates/ltp/generic-depthcharge-tftp-ramdisk-ltp-template.jinja2
```
so skipping it on staging for now.

File: test-configs.yaml [hunk 1]
```diff
@@ -165,6 +170,12 @@ test_plans:
     filters:
       - passlist: {defconfig: ['kselftest']}
 
+  ltp:
+    rootfs: debian_buster-ltp_nfs
+    pattern: 'ltp/{category}-{method}-{protocol}-nfs-ltp-template.jinja2'
```
Comment (gctucker) [inline test-configs.yaml:175]: With the suggestion above to use `root_type: nfs`
```suggestion
```

File: test-configs.yaml [hunk 2]
```diff
@@ -1961,6 +1972,7 @@ test_configs:
       - cros-ec
       - sleep
       - v4l2-compliance-uvc
+      - ltp
```
Comment (gctucker) [inline test-configs.yaml:1975]: Please keep test plan names in alphabetical order.

File: test-configs.yaml [hunk 3]
```diff
@@ -56,6 +56,11 @@ file_systems:
     type: debian
     ramdisk: 'buster-v4l2/20200821.0/{arch}/rootfs.cpio.gz'
 
+  debian_buster-ltp_nfs:
+    type: debian
+    ramdisk: 'buster-ltp/20200819/{arch}/initrd.cpio.gz'
+    nfs: 'buster-ltp/20200819/{arch}/full.rootfs.tar.xz'
```
Comment (gctucker) [inline test-configs.yaml:62]: If you add this:
```suggestion
    nfs: 'buster-ltp/20200819/{arch}/full.rootfs.tar.xz'
    root_type: nfs
```
then you shouldn't need to specify the template file pattern in the test plan definition.

**Review by gctucker — COMMENTED** [review]
Looks good, but the `pattern` attribute can actually be dropped.

**Comment by gctucker** [comment]
Removing the `staging-skip` tag as this should be working now.

File: test-configs.yaml (hunk 1 above)
Comment (Lakshmipathi) [reply test-configs.yaml]: Added above change

**Comment by gctucker** [comment]
Thanks, ultimately we should be using Linaro's test definitions and parameters in `test-definitions.yaml` to specify the list of tests to run.  For this PR, having one set of tests hard-coded such as `syscalls` so we can verify the rootfs is working in labs is a good step forward.

**Comment by Lakshmipathi** [comment]
LAVA job: https://lava.collabora.co.uk/scheduler/job/2601806

**Comment by gctucker** [comment]
Thanks, now let's see if we can get some jobs scheduled on staging.kernelci.org.

**Comment by gctucker** [comment]
Got one job on staging running with this PR, adding `staging-skip` now because it's conflicting with #487 and `kselftest` is nearer to getting merged.

**Comment by gctucker** [comment]
Some jobs were scheduled but there was a problem with the rootfs URL apparently:
https://lava.collabora.co.uk/scheduler/job/2602162

**Comment by Lakshmipathi** [comment]
updated rootfs url - it should work here after.

**Review by gctucker — APPROVED** [review]
Tested OK on staging: https://staging.kernelci.org/test/job/kernelci/branch/staging.kernelci.org/kernel/staging-20200915.1/plan/ltp/

**Pull request merged**